Remotion/Web/Core/UI/Controls/FormGridManager.cs

```diff
@@ -2146,13 +2148,13 @@ protected virtual void FormatTitleRow (FormGridRow titleRow)
 
       //  Adapt ColSpan for added markers column
       if (HasMarkersColumn)
-        titleRow.LabelsCell.ColSpan++;
+        titleRow.LabelsCell!.ColSpan++; // TODO: arg check
```

Comment:
LabelsCell and ControlsCell  etc can very likely move their checks into the property and make the property not-null.